Misc/NEWS.d/next/Core and Builtins/2018-11-13-01-03-10.bpo-32492.voIdcp.rst

```diff
@@ -0,0 +1,2 @@
+Speed up :class:`namedtuple` attribute access by 2.5x using a C fast-path
```

Comment:
Make comparison not with the current master, but with 3.7. There is a regression in master.

Reply:
```
Mean +- std dev: [old] 176 ns +- 2 ns -> [new] 110 ns +- 2 ns: 1.61x faster (-38%)
Significant (t=177.69)
```